NewForce-Cohort-5/tabloidmvc-the-spaghetti-squad
# Ticket # 23 View All User Profiles https://trello.com/c/TXJDiNqo

# Description

the user is an admin in the Tabloid application, when they select the User Profiles menu option they should be directed to the User Profiles list page and each user in the list should display the full name, the display name and the user type
And the list should be in ordered alphabetically by user display name.


## Type of change

Please delete options that are not relevant.


# Testing Instructions

`insert into UserProfile  (DisplayName, FirstName, LastName, Email, CreateDateTime, ImageLocation, UserTypeId)
VALUES ('aki', 'Aki', 'Endo', 'hi@here.net', '2022-03-01 11:15:45.730', NULL, 2);`

`insert into UserProfile  (DisplayName, FirstName, LastName, Email, CreateDateTime, ImageLocation, UserTypeId)
VALUES ('twy', 'J', 'Twy', 'j@email.come', '2022-03-01 11:15:45.730', NULL, 2);`


1.Please log in as the existing admin user admin@example.com
2. Navigate to USER PROFILES page
3. You should be able to view all existing users regardless of their User Type in alphabetical order based on "Display Name"
4. Log in as an Author user type using hi@here.net or j@email.come
5. Navigate to USER PROFILES page
6. The user should get a Not Found error message



# Checklist:
